prog/mutation.go

```diff
@@ -320,7 +320,28 @@ func (t *IntType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []*Ca
 }
 
 func (t *FlagsType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []*Call, retry, preserve bool) {
-	return mutateInt(r, s, arg)
+	if !t.BitMask {
```

Comment:
Please try to come up with some algorithm for !BitMask too.

Reply:
I am thinking that value of the argument should be valid so it would be enough to just choose randomly another value from the _t.Vals_ list?

Reply:
Well, most of the time.
But we still want some other values (combinations, completely random values, etc). E.g. some of the flags are both bitmask and enum, for example, bitmask in some lower bits and enum in higher bits. Sometimes we also have some flag values missing, because they are constantly added. Take a look at randGen.flags, I think we want something along these lines, maybe with updated probabilities, maybe with more consideration for BitMask value.